operations/helm/charts/grafana-agent/templates/hpa.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and (eq .Values.controller.type "deployment") .Values.controller.autoscaling.enabled }}
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "grafana-agent.fullname" . }}
+  labels:
+    {{- include "grafana-agent.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "grafana-agent.fullname" . }}
+  {{- with .Values.controller.autoscaling }}
+  minReplicas: {{ .minReplicas }}
+  maxReplicas: {{ .maxReplicas }}
+  metrics:
+    {{- with .targetCPUUtilizationPercentage }}
```

Comment:
Does it always make sense to scale based on both cpu and memory? If we scale by memory, but are not so concerned about cpu, can we do that?

Reply:
Yes, we can scale by only memory or CPU. The documentation is updated to reflect that `targetCPUUtilizationPercentage` or `targetMemoryUtilizationPercentage` can be diable by setting them to 0. Thanks!